File: app/src/main/java/com/gianessi/stargazers/models/UsersResponse.java
package com.gianessi.stargazers.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class UsersResponse {

    @SerializedName("total_count")
    private int totalCount;
    @SerializedName("incomplete_results")
    private boolean incompleteResults;
    @SerializedName("items")
    private List<User> items;

    public int getTotalCount() {
        return totalCount;
    }

    public boolean isIncompleteResults() {
        return incompleteResults;
    }

    public List<User> getItems() {
        return items;
    }
}

File: app/src/main/java/com/gianessi/stargazers/models/UsersResponse.kt
package com.gianessi.stargazers.models

import com.google.gson.annotations.SerializedName

class UsersResponse(
        @SerializedName("total_count") val totalCount: Int,
        @SerializedName("incomplete_results") val isIncompleteResults: Boolean,
        @SerializedName("items") val items: List<User>?)